internal/cmd/utils.go

```diff
@@ -131,23 +131,34 @@ func printTable(header []string, data [][]string) {
 	table.Render()
 }
 
-func destroyDatabase(client *turso.Client, name string) error {
+func destroyDatabases(client *turso.Client, names []string) error {
 	invalidateDatabasesCache()
 	invalidateDbTokenCache()
 	settings.PersistChanges()
 
-	start := time.Now()
-	s := prompt.Spinner(fmt.Sprintf("Destroying database %s... ", internal.Emph(name)))
+	destroy := func(name string) error {
+		start := time.Now()
+
+		if err := client.Databases.Delete(name); err != nil {
+			return err
+		}
+		elapsed := time.Since(start)
+
+		fmt.Printf("Destroyed database %s in %d seconds.\n", internal.Emph(name), int(elapsed.Seconds()))
```

Comment:
When this print is executed the spinner is still running with `Destroying databases...`.
This is a problem because all widgets from the `prompt` package expect that they are the only ones writing to `stdout` while they are active.
To make matters worse, when this assumption is broken the CLI gets to a weird state where it doesn't respond well - to fix it you need to either nuke the shell or reset it.

To keep things simple, my suggestion is to remove any printing logic from `destroy`, time the execution of the whole `errgroup`, and:
- If you deleted a single database, print: `fmt.Printf("Destroyed database %s in %d seconds.\n", internal.Emph(names[0]), int(elapsed.Seconds()))`
- if you deleted more than one database, print: `fmt.Printf("Destroyed %d databases in %d seconds.\n", len(names), int(elapsed.Seconds()))`

Reply:
Or do as you did on the are you sure prompt, joining their names. Be aware that the output can get weird if you are deleting too many databases.